feature/experimental/qos/ate_tests/mixed_sp_wrr_traffic_test/mixed_sp_wrr_traffic_test.go

```diff
@@ -638,354 +638,666 @@ func ConfigureQoS(t *testing.T, dut *ondatra.DUTDevice) {
 	dp3 := dut.Port(t, "port3")
 	d := &oc.Root{}
 	q := d.GetOrCreateQos()
+	if dut.Vendor() == ondatra.CISCO {
```

Comment:
The added code is lengthy and makes it difficult to read/review. Can we create a function, say ConfigureCiscoQoS(), to include all of these Cisco specific implementation so that we can:

switch dut.Vendor() {
case ondatra.CISCO:
  ConfigureCiscoQoS(.......)
default:
  ....
}

Reply:
As requested I have created a separate function on all the PRs raised.